src/stories/Profile/EditProfileInfo/index.stories.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import EditProfileView from "../../../components/Profile/ProfileInfoCard/editProfileDetailsModal";
+import ProviderWrapper from "../../../helpers/providerWrapper";
+
+export default {
+  title: "Profile/Edit Profile",
+  component: EditProfileView,
+};
+
+const Template = (args) => (
+  <ProviderWrapper>
+    <EditProfileView {...args} />
+  </ProviderWrapper>
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  profileData: {
+    country: "Argentina",
+    description: "",
+    displayName: "riju",
+    email: "sougatariju13@gmail.com",
+    following: ["rijusougata13"],
+    handle: "rijusougata31",
+    isEmpty: false,
+    isLoaded: true,
+    link_facebook: "",
+    link_github: "rijusougata13",
+    link_linkedin: "",
+    link_twitter: "",
+    orgFollowed: ["rijusougata13"],
+    organizations: (1)["testtest"],
+    photoURL:
+      "https://firebasestorage.googleapis.com/v0/b/buoyant-country-311408.appspot.com/o/user%2Frijusougata31%2Fimages%2Fnewfile?alt=media&token=2c295b90-2183-4d2b-99b9-b83c8e2ef9ac",
+    uid: "3Y7p7UVittajrIWWID7xF5MWDpt1",
+    website: "",
```

Comment:
Do not put your personal details as a component. Use some dummy data. Here you've already announced you email address publicly. Not good for you if you are a concerned person on privacy. As well as it is not a good practice to define personal details on a storybook component also.

Reply:
Thanks for mentioning this.I was not aware of this before that its a bad practice I am changing with dummy data.